lib/addons/addons.js

```diff
@@ -81,7 +83,7 @@ function transform_service(app, addon_definition, addon_plan, service) {
     },
     "provider_id": "alamo",
     "updated_at": (new Date(service.updated)).toISOString(),
-    "web_url": `https://${config.appkit_ui_url}/#/apps/${app.name}-${app.space}`
+    "web_url": `${config.appkit_ui_url}/#/apps/${app.name}-${app.space}`
   };
```

Comment:
A when in rome bug that I found in the UI/CLI itw as showing https://https://akkeris.octanner.io the https isn't necessary